### ISSUE TITLE
No such file issue in running ./prepare.sh on cluster for hadoop and spark

### ISSUE THREAD

**Issue by Aamer012** [intel-hadoop/HiBench#538]
I am facing issue in no such file while running ./prepare.sh. And even though i manually give the path of hdfs file to read and than run ./run.sh but still facing the issue. All the configuration settings i done according to the home page of hibench on github. Still facing issues. please guide me how can i configure hibench correctly for running my hadoop/spark and flink services. Help will be highly appreciated.

![screenshot from 2018-01-31 00-09-40](https://user-images.githubusercontent.com/32079521/35596914-41fb6cf4-061c-11e8-8c89-541d6ff6e63b.png)
![screenshot from 2018-01-31 00-10-12](https://user-images.githubusercontent.com/32079521/35596915-4217897a-061c-11e8-9324-760949733f93.png)

**Comment by carsonwang** [comment]
It looks like the input data was not generated successfully. You can take a look at the input directory on HDFS to see if there is any data. Wordcount just uses the utility `randomtextwriter` shipped with Hadoop to generate the input data. So you also try running that directly to make sure the configuration of Hadoop and HDFS are correct.

**Comment by Aamer012** [comment]
Thanks a lot carsonwang. Yes i manually put data in hdfs directory. path is hdfs://master:9000/data/test.txt and than i run directly ./run.sh but still have the same issue. Before i checked it for ./prepare.sh it also didnot work at all.  Path i mentioned is correct for ./run.sh. either i should change hibench.conf and workload conf for input/output path as well or it will work fine? I am working n it is not going on the rite way for 3 days.

**Comment by Aamer012** [comment]
@carsonwang  Please guide. Need your help dude

**Comment by carsonwang** [comment]
No, the data you manually put won't work. As you can see, prepare.sh just launch the hadoop job using the utility `randomtextwriter`. So I mean if this doesn't work, it's likely some configuration issues in your hadoop cluster instead of Hibench. You may check your Hadoop configuration first and make sure you can run any jobs.  
![image](https://user-images.githubusercontent.com/9278199/35684310-74d3c6a4-07a1-11e8-9b71-19e31e4d6ae8.png)

**Comment by Aamer012** [comment]
Dear carsonWag, My hadoop cluster is working fine as checked by running application on that with one master and two datanodes. But when i run HiBench according to instructions on github page i am unable to run it and i got same error no such file. I am sending you my configuration detail please tell me how can i run hibench . Its lot of time already spend but unable to get success.[
<img width="407" alt="1" src="https://user-images.githubusercontent.com/32079521/35768810-155c5076-0902-11e8-81a3-9a062c30c7af.png">
<img width="365" alt="2" src="https://user-images.githubusercontent.com/32079521/35768811-157cca86-0902-11e8-9830-956c796d644a.png">
<img width="411" alt="3" src="https://user-images.githubusercontent.com/32079521/35768812-15969efc-0902-11e8-873a-bcfe9cf963bf.png">


](url)

**Comment by Aamer012** [comment]
![screenshot from 2018-02-03 19-27-38](https://user-images.githubusercontent.com/32079521/35770280-70bff466-0918-11e8-9675-1de3477d82bf.png)

**Comment by Aamer012** [comment]
Now after much configurtaions and some more settings i m getting the error . @carsonwang  , guide me dude

**Comment by Tianny** [comment]
@Aamer012 
![image](https://user-images.githubusercontent.com/11434826/39753160-5efab672-52f0-11e8-9f11-0b6859cfe5b6.png)

double '/' here is wrong. Please check the $${hibench_home}/conf/hadoop.conf.
